.github/ISSUE_TEMPLATE/releases/patch-tuesday-release.md

```diff
@@ -5,13 +5,16 @@
 1. - [ ] Ensure all build agent scale sets have the appropriate number of machines allocated for a build
 1. - [ ] Merge any pending PRs or commits from `dev` branch:
       - [ ] &lt;placeholder link&gt;
-1. - [ ] Look up the latest [NuGet CLI versions](https://www.nuget.org/downloads) and update the NuGet entries of the [manifest.versions.json](https://github.com/microsoft/dotnet-framework-docker/blob/main/manifest.versions.json) file with the latest patch of each major/minor version listed in the file.
+1. - [ ] Look up the latest recommended [NuGet CLI version](https://www.nuget.org/downloads) and set the `nuget|version` variable to that version in the [manifest.versions.json](https://github.com/microsoft/dotnet-framework-docker/blob/main/manifest.versions.json) file.
 1. - [ ] Wait for latest cumulative updates (LCUs) to be released (typically at 10 AM PST on Patch Tuesday).
 1. - [ ] Gather list of KB numbers for the .NET Framework updates from the .NET Release team.
 1. - [ ] Look up the download URL for each of the KB numbers in [Microsoft Update Catalog](https://www.catalog.update.microsoft.com/) and input them into the [manifest.versions.json](https://github.com/microsoft/dotnet-framework-docker/blob/main/manifest.versions.json) file. If this is the first Patch Tuesday after the release of a new Windows version, you'll need to do the following for that version:
       - [ ] Add a new entry to the [manifest.versions.json](https://github.com/microsoft/dotnet-framework-docker/blob/main/manifest.versions.json) file to associate a URL for the new Windows version.
+1. - [ ] Update the datestamp variables in the [manifest.json file](https://github.com/microsoft/dotnet-framework-docker/blob/main/manifest.json). The datestamp for a particular image's tag should only be updated if any of the following conditions are met:
+      * The version of .NET Framework in the image has been serviced since the last release. (This includes servicing directly within the Dockerfile or from the base Windows image, see https://github.com/microsoft/dotnet-framework-docker/issues/783.)
```

Comment:
> or from the base Windows image

How is this determined?

Reply:
From step 5:

> Gather list of KB numbers for the .NET Framework updates from the .NET Release team.

Those KB numbers are associated with the Windows version that's being updated.